### PR TITLE
creates html skeleton

### PR DESCRIPTION
**Pull request by hhassan1230**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,78 +1,116 @@
...
\ No newline at end of file
+<!DOCTYPE html>
+  <html>
+    <head>
+      <meta charset="utf-8">
+      <title>Tech Job Academy</title>
+    </head>
+    <body>
+      <div class="hero">
+        <header>
```
Comment (sarony) [inline index.html]: Let's move hero inside of header.

File: index.html [hunk 2]
```diff
@@ -1,78 +1,116 @@
...
+          <img src="http://placehold.it/1410x350">
+          <nav>
```
Comment (sarony) [inline index.html:13]: `<div class="nav-bar"><div class="logo"></div><nav></nav></div>`

File: index.html [hunk 3]
```diff
@@ -1,78 +1,135 @@
...
\ No newline at end of file
+<!DOCTYPE html>
+  <html>
+    <head>
+      <meta charset="utf-8">
+      <title>Tech Job Academy</title>
+    </head>
+    <body>
+      <header>
+      <div class="nav-bar">
```
Comment (sarony) [inline index.html]: `<div class="logo"></div>`

File: index.html [hunk 4]
```diff
@@ -1,78 +1,135 @@
...
+        <nav>
+          <ul>
+            <li><a href="#" title="">OVERVIEW</a></li>
```
Comment (sarony) [inline index.html]: Fill out the titles with names of Nav

File: index.html [hunk 5]
```diff
@@ -1,78 +1,135 @@
...
+            <li><a href="#" title="">APPLICATION PROCESS</a></li>
+            <li><a href="#" title="">KEY DATES</a></li>
+          </ul>
+        </nav>  
+      </div>
+        <div class="hero">
+            <img src="http://placehold.it/1410x350">
+            <h1>Tech Job Academy</h1>
+            <p>An intensive, 16-week techincal training program preparing underemployed and unemployed NYC resident for in-demand tech jobs.</p>
+            <button type="">APPLY</button>
```
Comment (sarony) [inline index.html]: Let's add a link `<a href="#"></a>` inside button.

File: index.html [hunk 6]
```diff
@@ -1,78 +1,135 @@
...
+        </div>
+      </header>
+
+      <div class="overview">
+        <section>
```
Comment (sarony) [inline index.html]: Let's get rid of the div and add the `overview` class to the `section` tag.

File: index.html [hunk 7]
```diff
@@ -1,78 +1,135 @@
...
+          <h2>OVERVIEW</h2>
+          <p>Tech Jobs Academy is a public-private partnership between Microsoft, CUNY's City Tech, and the Tech Talent Pipline. This technical training program targets passionate underemployed and unemployed NYC resident ready to launch a new tech career.</p>
+          <p>Student will spend 16 weeks developing highly sought-after skills in cloud and server administration, and gaining career development skills to land a job upon program completion.</p>
```
Comment (sarony) [inline index.html]: Add `<div class="image-blurbs">` wrapping the four images

File: index.html [hunk 8]
```diff
@@ -1,78 +1,135 @@
...
+            <img src="http://placehold.it/140x100"><h3>Address</h3>
+            <div class="mail-signup">
+              <!-- Begin MailChimp Signup Form -->
+              <link href="//cdn-images.mailchimp.com/embedcode/slim-081711.css" rel="stylesheet" type="text/css">
+              <style type="text/css">
+                #mc_embed_signup{background:#fff; clear:left; font:14px Helvetica,Arial,sans-serif;  width:400px;}
+                /* Add your own MailChimp form style overrides in your site stylesheet or in this style block.
+                   We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
+              </style>
+              <div id="mc_embed_signup">
```
Comment (sarony) [inline index.html]: Let's nest the `form` under the `div`

File: index.html [hunk 9]
```diff
@@ -1,78 +1,135 @@
...
+                <label for="mce-EMAIL">Subscribe to our mailing list</label><br>
+                <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email address" required>
+                  <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+                  <div style="position: absolute; left: -5000px;"><input type="text" name="b_7b0ad64534cbb723ba1449cff_b7740812a4" tabindex="-1" value=""></div>
+                  <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
+                  </div>
+              </form>
+              </div>
+            </div>
+
```
Comment (sarony) [inline index.html]: Get rid of white space.

File: index.html [hunk 10]
```diff
@@ -1,78 +1,135 @@
...
+                  </div>
+              </form>
+              </div>
+            </div>
+            
+        </section>
+      </div>
+
+      <div class="curriculum">
+        <section>
```
Comment (sarony) [inline index.html]: Same class/section comment.

File: index.html [hunk 11]
```diff
@@ -1,78 +1,135 @@
...
+           <h2>CURRICULUM</h2>
```
Comment (sarony) [inline index.html]: Wrap these into their own `div`

File: index.html [hunk 12]
```diff
@@ -1,78 +1,135 @@
...
+           <img src="http://placehold.it/140x100"><h3>Curriculum Info 2</h3><p>Info blub</p>
+           <img src="http://placehold.it/140x100"><h3>Curriculum Info 3</h3><p>Info blub</p>
+           <img src="http://placehold.it/140x100"><h3>Curriculum Info 4</h3><p>Info blub</p>
+           <img src="http://placehold.it/140x100"><h3>Curriculum Info 5</h3><p>Info blub</p>
+           <img src="http://placehold.it/140x100"><h3>Curriculum Info 6</h3><p>Info blub</p>  
+        </section>
+      </div>
+
+      <div class="application">
+        <section>
```
Comment (sarony) [inline index.html]: Same class/section comment.

File: index.html [hunk 13]
```diff
@@ -1,78 +1,135 @@
...
+             </li>
+             <li>
+               <h3>Fill out application form</h3>
+               <p>The application helps us get to know you better. Answer questions about your interest in technology and what gets you excited about the program.</p>
+             </li>
+             <li>
+               <h3>Take entrace assessment</h3>
+               <p>Learn the fundamentals of networking and take this online entrace assessment to prove your knowledge.</p>
+             </li>
+           </ul>
```
Comment (sarony) [inline index.html]: This should be an `/ol``

File: index.html [hunk 14]
```diff
@@ -1,78 +1,135 @@
...
+           <img src="http://placehold.it/140x100"><h3>Curriculum Info 3</h3><p>Info blub</p>
+           <img src="http://placehold.it/140x100"><h3>Curriculum Info 4</h3><p>Info blub</p>
+           <img src="http://placehold.it/140x100"><h3>Curriculum Info 5</h3><p>Info blub</p>
+           <img src="http://placehold.it/140x100"><h3>Curriculum Info 6</h3><p>Info blub</p>  
+        </section>
+      </div>
+
+      <div class="application">
+        <section>
+           <h2>APPLICATION PROCESS</h2>
```
Comment (sarony) [inline index.html]: Let's wrap the `ol` in it's own div with class "application-steps"

File: index.html [hunk 15]
```diff
@@ -1,78 +1,135 @@
...
+             <li>
+               <h3>Fill out application form</h3>
+               <p>The application helps us get to know you better. Answer questions about your interest in technology and what gets you excited about the program.</p>
+             </li>
+             <li>
+               <h3>Take entrace assessment</h3>
+               <p>Learn the fundamentals of networking and take this online entrace assessment to prove your knowledge.</p>
+             </li>
+           </ul>
+           <h3>Eligibility Requirements</h3>
```
Comment (sarony) [inline index.html]: Let's wrap eligibility requirements with div with a class called "eligibility"

File: index.html [hunk 16]
```diff
@@ -1,78 +1,135 @@
...
+             <li>less than a 4-year degree</li>
+             <li>NYC resident</li>
+             <li>meet all federal selective service requirements</li>
+           </ul>
+          <p>Selective service link for information about selective service</p>
+          <button type="">APPLY</button> 
+        </section>   
+      </div>
+
+      <div class="keyDates">
```
Comment (sarony) [inline index.html]: Same section/class comment.

File: index.html [hunk 17]
```diff
@@ -1,78 +1,135 @@
...
+           <h3>Eligibility Requirements</h3>
+           <ul>
+             <li>at least 18 years old</li>
+             <li>less than $50K salary</li>
+             <li>less than a 4-year degree</li>
+             <li>NYC resident</li>
+             <li>meet all federal selective service requirements</li>
+           </ul>
+          <p>Selective service link for information about selective service</p>
+          <button type="">APPLY</button>
```
Comment (sarony) [inline index.html]: Add link inside button.

File: index.html (hunk 16 above)
Comment (sarony) [inline index.html]: Let's change the class name to "key-dates"

File: index.html [hunk 18]
```diff
@@ -1,78 +1,135 @@
...
+          <p>Selective service link for information about selective service</p>
+          <button type="">APPLY</button> 
+        </section>   
+      </div>
+
+      <div class="keyDates">
+        <section>
+           <h2>KEY DATES</h2>
+
+           <h3>Application Open</h3>
```
Comment (sarony) [inline index.html]: Wrap these in div with class "timeline-event"

File: index.html [hunk 19]
```diff
@@ -1,78 +1,135 @@
...
+           <h4>AUGUST 8</h4>
+
+           <h3>Application Close</h3>
+            &#8226;
+           <h4>AUGUST 28</h4>
+
+           <h3>Class Start</h3>
+            &#8226;
+           <h4>OCTOBER 5</h4>
+
```
Comment (sarony) [inline index.html:123]: Remove line break

File: index.html [hunk 20]
```diff
@@ -1,78 +1,135 @@
...
+        </section>
+      </div>
+      <div class="footer">
+        <section>
```
Comment (sarony) [inline index.html]: same section/class comment.

File: index.html [hunk 21]
```diff
@@ -1,78 +1,135 @@
...
+           <footer>
+             <p>Microsoft</p>
```
Comment (sarony) [inline index.html]: These should be `img` tags with links.

File: index.html [hunk 22]
```diff
@@ -1,78 +1,138 @@
...
+<!DOCTYPE html>
+  <html>
+    <head>
+      <meta charset="utf-8">
+      <title>Tech Job Academy</title>
+    </head>
+    <body>
+      <header>
+      <div class="nav-bar">
+        <div class="logo">
```
Comment (hhassan1230) [inline index.html]: add div class logo

File: index.html [hunk 23]
```diff
@@ -1,78 +1,151 @@
...
+            <h1>Tech Job Academy</h1>
+            <p>An intensive, 16-week techincal training program preparing underemployed and unemployed NYC resident for in-demand tech jobs.</p>
+            <button><a href="#" title="APPLY">APPLY</a></button>
+        </div>
+      </header>
+      <section class="overview">
+        <h2>OVERVIEW</h2>
+        <p>Tech Jobs Academy is a public-private partnership between Microsoft, CUNY's City Tech, and the Tech Talent Pipline. This technical training program targets passionate underemployed and unemployed NYC resident ready to launch a new tech career.</p>
+        <p>Student will spend 16 weeks developing highly sought-after skills in cloud and server administration, and gaining career development skills to land a job upon program completion.</p>
+          <div class="image-blurbs">
```
Comment (sarony) [inline index.html]: Add p tags for the image blurbs

File: index.html [hunk 24]
```diff
@@ -1,78 +1,151 @@
...
+              </form>
+            </div>
+          </div> 
+      </section>
+    
+
+      
+      <section class="curriculum">
+         <h2>CURRICULUM</h2>
+         <div class="image-blurbs">
```
Comment (sarony) [inline index.html]: Chane "image-blurbs" to "image-blurb"

File: index.html [hunk 25]
```diff
@@ -1,78 +1,151 @@
...
+             <li>less than a 4-year degree</li>
+             <li>NYC resident</li>
+             <li>meet all federal selective service requirements</li>
+           </ul>
+         </div>
+        <p>Selective service link for information about selective service</p>
+        <button><a href="#" title="APPLY">APPLY</a></button>
+      </section>    
+      <section class="keyDates">
+        <div class="timeline-event">
```
Comment (sarony) [inline index.html]: Should be "timeline"

File: index.html [hunk 26]
```diff
@@ -1,78 +1,151 @@
...
+             <li>less than $50K salary</li>
+             <li>less than a 4-year degree</li>
+             <li>NYC resident</li>
+             <li>meet all federal selective service requirements</li>
+           </ul>
+         </div>
+        <p>Selective service link for information about selective service</p>
+        <button><a href="#" title="APPLY">APPLY</a></button>
+      </section>    
+      <section class="keyDates">
```
Comment (sarony) [inline index.html]: Change to "key-dates"

File: index.html [hunk 27]
```diff
@@ -1,78 +1,151 @@
...
+        <div class="timeline-event">
+           <h2>KEY DATES</h2>
+
+           <h3>Application Open</h3>
```
Comment (sarony) [inline index.html]: Need to wrap these in "key-date"

**Pull request merged**